style.css

```diff
@@ -0,0 +1,102 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  max-width: 480px;
+  font-family: "Poppins", sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background: hsl(20, 3%, 23%);
+  width: 100%;
+  height: 48px;
```

Comment:
- [ ] I will recommend you increase the `height` of the header align the `logo` and `hamburger` respectively to look same like the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
Your design 
![image](https://user-images.githubusercontent.com/62849872/143556911-66d98dec-acb9-4547-b172-2267feffd0da.png)

Figma design 
![image](https://user-images.githubusercontent.com/62849872/143557053-37601c38-c389-4363-b9f7-fc775d490f7c.png)

Reply:
I have effected these changes